.github/workflows/backend_checks.yml

```diff
@@ -256,6 +256,7 @@ jobs:
 
     runs-on: ubuntu-latest
     timeout-minutes: 45
+    continue-on-error: true
```

Comment:
this was previously removed because it caused tests that timed out to pass. Instead they should fail on timeout. More context here- https://ethyca.slack.com/archives/CBKK8TS74/p1733926881926369. Why did we need to add this back?

Reply:
I removed it! I was seeing some weird behavior earlier where a test failures caused the rest of the test suite to be skipped